src/components/mainContent/Navbar.css

```diff
@@ -0,0 +1,40 @@
+.dropdown {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 450px;
+}
+
+.dropdown-content {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 0 0 10px 10px;
+  display: none;
+  position: absolute;
+  background-color: #f7f7f7;
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px 898F9C;
+  z-index: 1;
+  text-align: center;
+  top: 46px;
+  left: -63px;
```

Comment:
Same comment on arbitrary numbers!

Reply:
When I implemented the drop down originally, it stuck right to the array and went off screen. In order to position the dropdown right under the navbar and center it with the button I had to experiment with odd values. Is this an incorrect solution to the issue?

Reply:
Hm, I'm generally adverse to magic numbers but this may also become a problem once we implement a responsive design. We can punt this to later, but I think we should put a comment here that explains why the numbers are there + maybe a "TODO" to fix it.